packaging_ignore.yml

```diff
@@ -10,6 +10,9 @@ base:
   - ".* warning: overriding recipe for target 'check'"
   - "dpkg-genchanges: warning: the current version .* is earlier than the previous one .*"
   - ".*: warning: '__s' may be used uninitialized in this function \\[-Wmaybe-uninitialized\\].*"
+  - ".* unused substitution variable \\$\\{misc:Depends\\}.*"
```

Comment:
can't we just remove `${misc:Depends}` from `Depends:`, as in https://github.com/citusdata/packaging/commit/fd537ba0dd3736512a337a281fd9aaf52a6b862a ?

Btw, what does having or not having `${misc:Depends}` in `Depends :` line mean ? Is there any docs that you can share here so that I can better understand ?

Reply:
I am ok with this change. The `${misc:Depends} may be helpful in case `dh_*` scripts actually find some dependencies that should be added to the dependencies. We may observe some empty lists from time to time, but that's ok I guess.